youtube_dl/extractor/bbc.py

```diff
@@ -1006,12 +1008,13 @@ def _real_extract(self, url):
             version_id = clip_data.get('versionID')
             if version_id:
                 title = smp_data['title']
+                # also try for higher resolutions
+                self._MEDIA_SETS.insert(0, 'iptv-all')
```

Comment:
as the `iptv-all` gives better quality in general, just drop `_MEDIA_SETS` from the extractor and just use the default value from `BBCCoUkIE`.